src/test/clojure/cemerick/pomegranate/aether_test.clj

```diff
@@ -330,6 +330,8 @@
      :local-repo tmp-local-repo-dir))
   (is (= 3 (count (.list (io/file tmp-local-repo-dir "group" "artifact" "1.0.0"))))))
 
+(java.lang.System/setProperty "aether.checksums.forSignature" "true")
```

Comment:
As written [here](https://issues.sonatype.org/plugins/servlet/mobile#issue/NEXUS-8213), this property makes aether write checksum files for everything again.